src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	
+	/**
```

Comment:
Good: Comment for class added. Note that you should place these comments on top of the class instead of inside.